internal/cli/default_setter_opts.go

```diff
@@ -389,7 +389,6 @@ func (*DefaultSetterOpts) DefaultQuestions() []*survey.Question {
 			Prompt: &survey.Select{
 				Message: "Default Output Format:",
 				Options: []string{plaintextFormat, jsonFormat},
-				Default: config.Output(),
```

Comment:
plaintextFormat ("plaintext") is the default one and I think we should use it here too.
@gssbzn wdyt about updating config.Output() to return plaintextFormat instead of an empty string, when the output flag is not set?
we are already doing something similar with TelemetryEnabled:
```
func (p *Profile) TelemetryEnabled() bool {
	return isTelemetryFeatureAllowed() && p.GetBoolWithDefault(TelemetryEnabledProperty, true)
}

```

Reply:
this is a bit of a legacy decision where we decided not to really name the plaintext output, the risk here is we need to make this backward compatible, some configs will have `output = ''` some users may be using `-o=` to signal plain text so those need to still work

Reply:
we looked a bit more into having a default value for output in config, but making sure we're keeping the backward compatibility is a bit tricky (and a few tests failed) and would increase the scope of this task.

as an alternative, what we're thinking is to have the check in this survey function (and not affect any other flows), and if the output is not set in the profile, consider plaintext as the default option.
This will keep the existing behavior for the scenario when the output is already set in the profile (e.g. if json is set in the profile, then it will be selected automatically in this survey too).

Reply:
I've pushed the proposed change.